app/helpers/wcrypt.js

```diff
@@ -0,0 +1,4 @@
+const bcrypt = require('bcryptjs');
+const { NUM_SALT } = require('../../config/constants');
+
+exports.encryptPassword = (value, salt = NUM_SALT) => bcrypt.hashSync(value, salt);
```

Comment:
Here you can also do the same of adding a custom error, this more than anything to have better error tracking.